### ISSUE TITLE
Please consider adding a "no legacy boot" option

### ISSUE THREAD

**Issue by GrahamKnap** [WoeUSB/WoeUSB#109]
The "legacy boot" step of the media creation process can take a long time:

```
Info: Installing GRUB bootloader for legacy PC booting support...
Installing for i386-pc platform.
Installation finished. No error reported.
```

If this step isn't required to boot the Windows installation on systems that support UEFI, then it would be helpful if you could add an option that skips this step.

For Debian users, this could also avoid the need to install `grub-pc-bin`.

**Comment by RisaKirisu** [comment]
Indeed. This step takes ridiculous long to finish. A simple fix is to just comment out line 220-227.

**Comment by ghost** [comment]
true. I've been waiting for an hour almost

**Comment by JojiiOfficial** [comment]
Whats wrong with the Options->Skip legacy grub bootloader option?

**Comment by simafrus** [comment]
> Whats wrong with the Options->Skip legacy grub bootloader option?

cli hasnt this option
